### PR TITLE
Contribution guide

### LINKED ISSUE #50: ✨ [Feature request]: Add doc for how to contribute
### Is your feature request related to a problem? Please describe.

We are getting great contributions, however, one common question we get asked is "How to contribute".

### Describe the solution you'd like.

Add a doc for "How to contribute", which provides step by step guide for contributing to this site.

### Describe alternatives you've considered.

NA

### Screenshot / Screenshare

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by joypoddar**
Kindly read thoroughly and suggest changes.

Closes #50

### CODE REVIEW

**Review by atapas — CHANGES_REQUESTED** [review]
Suggested Changes.

File: docs/How-To-Guides/Contribute/contribute-to-docs.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+New to open-source? Follow [this guide](https://opensource.guide/how-to-contribute/) to jumpstart your Open Source journey 🚀
+:::
+
+## Getting started
+
+Contribution marks the beginning of our first steps in growing together. Below is a list of probable ways in which you can contribute to the [reactplay/docs](https://github.com/reactplay/docs) repo.
+
+- write documentation for various parts of ReactPlay
+- suggest changes to the existing docs
+- improve the documentation
```
[the remark this answers is not in the document]
Comment (joypoddar) [reply docs/How-To-Guides/Contribute/contribute-to-docs.md:25]: > So this doc should talk about the over all contributions that some one can make on ReactPlay. Not on the docs project alone.
> 
> So Rephrase it and add things like,
> 
> * Create Plays(Link to Create play doc)
> * Do Feature enhancements(Link to the feature create)
> * Test and Find Bug(Link to bugs create)
> * Improve docs(Link to the doc repo)
> * Manage our social handles
> * Give us the feedback, suggestions, help improving our UX
> * Be an advocate of ReactPlay and help us arranging events, getting sponsorship and support.
> * Write articles on our blog

Great suggestion. I can add a lot of things mentioned above. 

So, regarding what you said about this doc serving as a general contribution guide, I had the same question. please refer to [this](https://github.com/reactplay/docs/issues/50#issuecomment-1498874608) and let me know what to do.

File: src/css/custom.css [hunk 2]
```diff
@@ -28,3 +28,12 @@
   --ifm-color-primary-lightest: #4fddbf;
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.3);
 }
+
+.first-issue {
```
Comment (Sachin-chaurasiya) [inline src/css/custom.css:32]:
```suggestion
.good-first-issue-badge {
```

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

File: docs/contribution-guide.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- You could give us feedback, suggestions and help us improve our UX.
+- Have interest in interacting with public, you are welcome to manage our social media handles like [Twitter](https://twitter.com/ReactPlayIO) and [LinkedIn](https://www.linkedin.com/company/reactplay/).
+- Have a flair for writing, you could publish articles on our [blog](https://blog.reactplay.io/).
+- You could also be our advocate and help us in arranging events, sponsors and support.
+
+You as a contributor could just about do anything mentioned above or any combination of the things above. We love our contributors and do our best to accommodate what you bring to the table.
+
+:::tip
+If you are a beginner look for the <span className="first-issue">good first issue</span> label in the issues page.
+:::
```
Comment (Sachin-chaurasiya) [inline docs/contribution-guide.md:44]: Can we update the className here as well?

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

File: src/css/custom.css [hunk 4]
```diff
@@ -28,3 +28,19 @@
...
+  font-size: 0.875rem; /* 14px */
+  line-height: 1.25rem; /* 20px */
+}
+
+.ionicon {
+  display: inline-block;
+  width: 1.3rem;
+  height: 1.3rem;
+  margin-right: 4px;
+}
```
Comment (Sachin-chaurasiya) [inline src/css/custom.css:46]: @joypoddar , are we using this class somewhere?

**Review by Sachin-chaurasiya — APPROVED** [review]
Great work @joypoddar  🙌🏻,
Thanks for the contributions ✨

**Review by atapas — APPROVED** [review]

**Pull request merged**